### PR TITLE
update RST format

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

File: docs/contributing/contributing.rst [hunk 1]
```diff
@@ -45,14 +45,16 @@ Finding an Issue
 You can familiarize yourself with the Mautic contribution process by reviewing the list of **issues** on the :xref:`Mautic Docs issues` section. These issues have a relatively limited scope. 
 
 After you have selected an issue on the :xref:`Mautic Docs issues` page, follow the below steps:
- 1. Add a comment indicating that you would like to own the ticket. This is to avoid conflicts with others also working on the issue.
- 2. After a Mautic administrator assigns you the issue, you can modify files and track changes on GitHub using command line utility, your web browser, GitHub Desktop, or Gitpod.
- 3. Submit a pull request, and the Mautic administrator will review your changes.
+
+1. Add a comment indicating that you would like to own the ticket. This is to avoid conflicts with others also working on the issue.
+2. After a Mautic administrator assigns you the issue, you can modify files and track changes on GitHub using command line utility, your web browser, GitHub Desktop, or Gitpod.
```
Comment (RCheesley) [inline docs/contributing/contributing.rst:50]: Please fix the Vale errors - need to add Gitpod to the accept.txt file as it doesn't recognise it.

File: docs/contributing/contributing.rst [hunk 2]
```diff
@@ -45,14 +45,16 @@ Finding an Issue
...
+3. Submit a pull request, and the Mautic administrator will review your changes.
```
Comment (RCheesley) [inline docs/contributing/contributing.rst:51]:
```suggestion
3. Once you submit a pull request, an admin reviews your changes.
```

File: docs/contributing/contributing.rst [hunk 3]
```diff
@@ -45,14 +45,16 @@ Finding an Issue
...
 
 Creating an Issue or Proposing Changes
 *************************************
 
 You can create an issue or propose changes by following these steps:
+
 1. Create a new GitHub issue associated with the relevant repository and propose your change there. Be sure to include implementation details and the rationale for the proposed change.
 2. The submitted issue will be reviewed by the Mautic administrator.
```
Comment (RCheesley) [inline docs/contributing/contributing.rst:59]:
```suggestion
2. A Mautic team member reviews all submitted issues.
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
A few changes - also make sure you have the Vale linter installed and running, and check for the errors that it flags :)

**Comment by fakela** [comment]
Another contributor has created a PR to add the # numbered list format to the other sections that were numbered 1,2,3

**Comment by fakela** [comment]
vale fixed 😁

File: docs/contributing/contributing.rst [hunk 4]
```diff
@@ -45,17 +45,19 @@ Finding an Issue
...
 *************************************
 
 You can create an issue or propose changes by following these steps:
-1. Create a new GitHub issue associated with the relevant repository and propose your change there. Be sure to include implementation details and the rationale for the proposed change.
-2. The submitted issue will be reviewed by the Mautic administrator.
-3. If the Mautic administator approves the issue you may start working on the task
+
+#. Create a new GitHub issue associated with the relevant repository and propose your change there. Be sure to include implementation details and the rationale for the proposed change.
+#. A Mautic team member reviews all submitted issues.
+#. Once the issue is approved, you may start working on it.
```
Comment (RCheesley) [inline docs/contributing/contributing.rst:60]: Vale suggestion

```suggestion
#. Once approved, you may start working on the issue.
```

**Review by RCheesley — COMMENTED** [review]
Strange that the Gitpod link is not being added to the description but I am testing using the direct URL gitpod.io/#https://github.com/mautic/user-documentation/pull/53/files.  I haven't got time to do a full review at this moment so this is just some of the big things I spotted.

There is an issue here with the code being outside of the code area and with most of the code blocks below. It looks to me like incorrect indentation - maybe a copy/paste error?

Preview contributing rst — user-documentation — Gitpod Code

There are several changes that I had made before which have sadly been overwritten it seems, a few I spotted:

Line 69 does not make sense - 'submit a PR link with Mautic reviewers'?

Line 71 is incorrect - 'Git requires a GitHub user account' - GitHub requires a GitHub account. Git does not (you can use any platform that supports the software Git).

The images I think need to be wider - it is difficult to see them all.

We have a LOT of RST errors to fix, some of which I thought I had addressed with my PR a while back but they are flagging up a failed test.

Take a look at the failing GitHub Action: https://github.com/mautic/user-documentation/runs/5988230445?check_suite_focus=true - maybe these need fixing outside of this PR before we continue fixing the issues with this page.

**Comment by fakela** [comment]
> Strange that the Gitpod link is not being added to the description but I am testing using the direct URL https://gitpod.io/#https://github.com/mautic/user-documentation/pull/53/files. I haven't got time to do a full review at this moment so this is just some of the big things I spotted. Please ensure that all PRs are checked with Gitpod as it is very easy to miss things!
> 
> There is an issue here with the code being outside of the code area and with most of the code blocks above it. It looks to me like incorrect indentation - maybe a copy/paste error?
> 
> Preview contributing rst — user-documentation — Gitpod Code
> 
> There are several changes that I had made before which have sadly been overwritten it seems, a few I spotted:
> 
> Line 69 does not make sense - 'submit a PR link with Mautic reviewers'?
> 
> Line 71 is incorrect - 'Git requires a GitHub user account' - GitHub requires a GitHub account. Git does not (you can use any platform that supports the software Git).
> 
> The images I think need to be wider - it is difficult to see them all.
> 
> We have a LOT of RST errors to fix, some of which I thought I had addressed with my PR a while back but they are flagging up a failed test.
> 
> Take a look at the failing GitHub Action: https://github.com/mautic/user-documentation/runs/5988230445?check_suite_focus=true - maybe these need fixing outside of this PR before we continue fixing the issues with this page.

Someone is also contributing to this page and I decided to just fix the immediate stuff I wanted and wait for the person who was fixing the other stuff

**Comment by fakela** [comment]
> Strange that the Gitpod link is not being added to the description but I am testing using the direct URL https://gitpod.io/#https://github.com/mautic/user-documentation/pull/53/files. I haven't got time to do a full review at this moment so this is just some of the big things I spotted. Please ensure that all PRs are checked with Gitpod as it is very easy to miss things!
> 
> There is an issue here with the code being outside of the code area and with most of the code blocks above it. It looks to me like incorrect indentation - maybe a copy/paste error?
> 
> Preview contributing rst — user-documentation — Gitpod Code
> 
> There are several changes that I had made before which have sadly been overwritten it seems, a few I spotted:
> 
> Line 69 does not make sense - 'submit a PR link with Mautic reviewers'?
> 
> Line 71 is incorrect - 'Git requires a GitHub user account' - GitHub requires a GitHub account. Git does not (you can use any platform that supports the software Git).
> 
> The images I think need to be wider - it is difficult to see them all.
> 
> We have a LOT of RST errors to fix, some of which I thought I had addressed with my PR a while back but they are flagging up a failed test.
> 
> Take a look at the failing GitHub Action: https://github.com/mautic/user-documentation/runs/5988230445?check_suite_focus=true - maybe these need fixing outside of this PR before we continue fixing the issues with this page.

This page is an important page as it helps with onboarding so I think we can merge this while others PRs are created for the issues listed out. I will suggest that creating PRs that just fixes a particular issue or section especially when the page is here already and we are just updating might be easier to manage and review.

And for the RST issues it's in a lot of pages which we will continue to fix and manage as we go by.

I will create a PR for some of the RST errors in that regard.

**Pull request merged**